lib/Util/Notices/PermanentNotices.php

```diff
@@ -32,7 +32,11 @@ public function init() {
       Menu::isOnMailPoetAdminPage()
       && $_GET['page'] !== 'mailpoet-welcome-wizard'
     );
-    $this->discounts_announcement->init(empty($_GET['page']) && is_admin());
+    $this->discounts_announcement->init(
+      empty($_GET['page'])
+      && is_admin()
+      && strpos($_SERVER['REQUEST_URI'], 'wp-admin/index.php') !== false
```

Comment:
This works when the URL is `/wp-admin/index.php` but does not when it's `/wp-admin/`